UnityProject/Assets/Scripts/Objects/Doors/DoorMasterController.cs

```diff
@@ -522,15 +513,7 @@ public void Close(bool byForce = false)
 
 		public void Open(bool byForce = false)
 		{
-			if (IsClosed == false) return;
-
-			if (isFireLock == false)
```

Comment:
Can you test and make sure that firelocks aren't broken from this change?

You can also test if it's working on multiplayer by using this: https://unitystation.github.io/unitystation/development/Multiplayer-PlayMode/